src/module/party/party-sheet.js

```diff
@@ -119,11 +119,11 @@ export default class OsePartySheet extends FormApplication {
 
   _recursiveAddFolder(folder) {
     folder.contents.forEach((actor) => this._addActorToParty(actor));
-    folder.children.forEach((folder) => this._recursiveAddFolder(folder));
+    folder.children.forEach((folder) => this._recursiveAddFolder(folder.folder));
   }
 
   async _onDropFolder(event, data) {
-    if (data.documentName !== "Actor") {
+    if (data.documentName !== "Folder") {
```

Comment:
I don't think this is correct. The documentName for a folder in the actor subdirectory is always Actor. This line is here to ignore folders from say, the Item subdirectory, or what have you

Reply:
Indeed, and this is what broke the changes :) 

New commit should fix this behaviour!